content/patterns/spinbutton/examples/quantity-spinbutton.html

```diff
@@ -0,0 +1,415 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quantity Spin Button Example</title>
+
+    <!-- Core JS and CSS shared by all examples -->
+    <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+    <link rel="stylesheet" href="../../../shared/css/core.css">
+    <script src="../../../shared/js/examples.js"></script>
+    <script src="../../../shared/js/highlight.pack.js"></script>
+    <script src="../../../shared/js/app.js"></script>
+    <script data-skipto="colorTheme:aria; displayOption:popup; containerElement:div" src="../../../shared/js/skipto.js"></script>
+
+    <!-- JS and CSS for this example -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
+    <link href="css/quantity-spinbutton.css" rel="stylesheet">
+    <script src="js/quantity-spinbutton.js"></script>
+  </head>
+  <body>
+    <nav aria-label="Related Links" class="feedback">
+      <ul>
+        <li><a href="https://github.com/orgs/w3c/projects/143">Related Issues</a></li>
+        <li><a href="../spinbutton-pattern.html">Design Pattern</a></li>
+      </ul>
+    </nav>
+    <main>
+      <h1>Quantity Spin Button Example</h1>
+
+      <section>
+        <h2>About This Example</h2>
+        <img alt="" class="example-page-example-icon" src="../../../images/pattern-spinbutton.svg">
+        <p>
+          The following example uses the <a href="../spinbutton-pattern.html">Spin Button Pattern</a> to implement three quantity inputs.
+        </p>
+        <p>Similar examples include:</p>
+        <ul>
+          <li><a href="../../toolbar/examples/toolbar.html">Toolbar Example</a>: A toolbar that contains a spin button for setting font size.</li>
+        </ul>
+      </section>
+
+      <section>
+        <div class="example-header">
+          <h2 id="ex_label">Example</h2>
+        </div>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div id="example">
+          <div class="spinners">
+            <fieldset>
+              <legend>Guests</legend>
+              <div class="spinner-fields">
+                <div class="spinner-field">
+                  <label for="adults">
+                    Adults
```

Comment:
Visually looking good to me! Nice work!! 

One thought: previously with the date picker a user knows there's a certain amount of days in a month and all the month options, but in this example we don't know the max/min of guests until we increase/decrease. Maybe we can update the labels to be `Adults (0-8)`, `Kids (0-8)`, and `Animals (0-12)`